stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/state/MessageAction.kt

```diff
@@ -61,6 +61,11 @@ public class Copy(message: Message) : MessageAction(message)
  */
 public class Edit(message: Message) : MessageAction(message)
 
+/**
+ * Start editing moderated message
```

Comment:
Missing full stop at the end

Reply:
In the end, we might be able to just remove this option as there shouldn't really be a difference in editing a moderated message vs a regular one